### PR TITLE
Fixed code style and added more strict typing to `SystemInfoDumpCommand`

### PR DESCRIPTION
**Pull request by konradoboza**
I noticed some failing CS and some nitpicks in the affected Command.

### CODE REVIEW

**Review by webhdx — APPROVED** [review]

**Review by alongosz — COMMENTED** [review]
@konradoboza does this apply to `main` only?

**Comment by konradoboza** [comment]
It seems `4.2` is also affected. Rebased, thanks for pointing that out @alongosz.

File: .github/workflows/browser-tests.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
         uses: ibexa/gh-workflows/.github/workflows/browser-tests.yml@main
         with:
             project-edition: 'oss'
-            project-version: '4.2.x-dev'
+            project-version: '4.3.x-dev'
```
Comment (alongosz) [inline .github/workflows/browser-tests.yaml:16]: You can skip it now ;)

File: composer.json [hunk 2]
```diff
@@ -31,14 +31,14 @@
...
-        "ibexa/search": "~4.2.0@dev",
-        "ibexa/http-cache": "~4.2.0@dev",
+        "ibexa/doctrine-schema": "~4.3.0@dev",
+        "ibexa/content-forms": "~4.3.0@dev",
+        "ibexa/design-engine": "~4.3.0@dev",
+        "ibexa/user": "~4.3.0@dev",
+        "ibexa/fieldtype-richtext": "~4.3.0@dev",
+        "ibexa/rest": "~4.3.0@dev",
+        "ibexa/search": "~4.3.0@dev",
+        "ibexa/http-cache": "~4.3.0@dev",
```
Comment (alongosz) [inline composer.json:41]: Seems like this rebase didn't go so well. I feel responsible for making the fuss :sweat_smile:

**Review by alongosz — CHANGES_REQUESTED** [review]

**Review by alongosz — APPROVED** [review]

**Review by Steveb-p — APPROVED** [review]

**Comment by konradoboza** [comment]
Merged up via https://github.com/ibexa/system-info/commit/cca60172d907368cb24797a5b1c1b3f90baed9e0.

**Pull request merged**